src/sidebar/components/Annotation/AnnotationShareControl.js

```diff
@@ -123,53 +142,69 @@ function AnnotationShareControl({
   );
 
   return (
-    <div className="AnnotationShareControl" ref={shareRef}>
+    <div className="relative" ref={shareRef}>
       <IconButton
         icon="share"
         title="Share"
         onClick={toggleSharePanel}
         expanded={isOpen}
       />
       {isOpen && (
-        <div className="annotation-share-panel">
-          <div className="annotation-share-panel__header">
-            <div className="annotation-share-panel__title">
+        <Card
+          classes={classnames(
+            // Prefer width 96 (24rem) but ensure that component isn't wider
+            // than 85vw
+            'w-96 max-w-[85vw]',
+            // Position this Card above its IconButton. Account for larger
+            // IconButtons in touch interfaces
+            'absolute bottom-8 right-1 touch:bottom-touch-minimum',
+            'space-y-2 p-2',
+            // Cards do not have a border in the clean theme. Turn it back on.
+            'theme-clean:border'
+          )}
+        >
+          <div className="flex items-center pb-2 border-b">
+            <h2 className="text-brand text-lg font-medium">
```

Comment:
Again, TW's text- and font-related class naming is sometimes annoying. `text-brand` is a color, `text-lg` is a size and `font-medium` is a weight.

Reply:
Presumably there is nothing to stop us adding a prefix to color names like `brand` if their color-ness is non-obvious?

Reply:
Indeed. I've already done this in the case of "state colors". When I started with TW, I defined some state-related colors as follows (in tailwind configuration):

```
colors: {
  // ...
  success: '#00a36d',
  notice: '#fbc168',
  error: '#d93c3f',
}
```

but that led to classnames like `bg-error` and `text-success`, which were ambiguous. They've since been altered to:

```
green: {
  success: '#00a36d',
},
yellow: {
  notice: '#fbc168',
},
red: {
  error: '#d93c3f',
},
```

Such that we have `bg-red-error` and `text-green-success` (that's how TW handles nested color-name definitions). If we continue to find `brand` too ambiguous, we could make it `red-brand`.

Technical note: this colors object is _extending_ Tailwind's available colors, so it's _adding_ new entries for green, yellow, red. [Tailwind provides a whole bumper crop of colors out of the box](https://tailwindcss.com/docs/customizing-colors), though those don't manifest (i.e. create any CSS rules) unless we use them in source.

Reply:
(I think in a way all of this is part of tailwind's "design", nudging us away from too much abstraction).